Neos.FluidAdaptor/Tests/Unit/ViewHelpers/Form/CheckboxViewHelperTest.php

```diff
@@ -51,12 +51,14 @@ protected function setUp(): void
      */
     public function renderCorrectlySetsTagNameAndDefaultAttributes()
     {
-        $this->mockTagBuilder->expects(self::any())->method('setTagName')->with('input');
-        $this->mockTagBuilder->expects(self::at(2))->method('addAttribute')->with('type', 'checkbox');
-        $this->mockTagBuilder->expects(self::at(3))->method('addAttribute')->with('name', 'foo');
-        $this->viewHelper->expects(self::once())->method('registerFieldNameForFormTokenGeneration')->with('foo');
-        $this->mockTagBuilder->expects(self::at(4))->method('addAttribute')->with('value', 'bar');
+        $this->mockTagBuilder->expects(self::atLeastOnce())->method('setTagName')->with('input');
```

Comment:
That was `any()` before?

Reply:
Yes, but the test name contains `SetsTagName` and `any()` does not check that. So it was wrong before…